std-bits/table/src/main/java/org/enso/table/data/column/operation/unary/TruncatedTimePartOperation.java

```diff
@@ -0,0 +1,36 @@
+package org.enso.table.data.column.operation.unary;
+
+import org.enso.table.data.column.builder.LongBuilder;
+import org.enso.table.data.column.operation.UnaryOperation;
+import org.enso.table.data.column.operation.map.MapOperationProblemAggregator;
+
+import java.time.temporal.ChronoField;
+import java.time.temporal.Temporal;
+import java.time.temporal.TemporalField;
+
+public class TruncatedTimePartOperation extends DatePartOperation {
+  public static final String MICROSECOND = "microsecond";
+  public static final UnaryOperation MICROSECOND_INSTANCE =
+      new TruncatedTimePartOperation(MICROSECOND, ChronoField.MICRO_OF_SECOND, 1000);
+
+  public static final String NANOSECOND = "nanosecond";
+  public static final UnaryOperation NANOSECOND_INSTANCE =
+      new TruncatedTimePartOperation(NANOSECOND, ChronoField.NANO_OF_SECOND, 1000);  private final int truncation;
+
+  private TruncatedTimePartOperation(String name, TemporalField field, int truncation) {
+    super(name, field, false);
```

Comment:
```suggestion
    super(name, field, true);
```

Shouldn't `timeField = true` here?

Reply:
yes - fixed already!